entries/h/hpe.com.json

```
@@ -0,0 +1,20 @@
+{
+  "HPE": {
```

```suggestion
  "Hewlett Packard Enterprise": {
```
I would use the full business name as 'HPE' is more of an "also know as" name

---

Thanks! 😃